docs/index.md

```diff
@@ -1,55 +1,17 @@
 # Internal - Build and Release Process for a New Version of the Docs and Index for Devdocs
```

Comment:
Who will be maintaining this index file? I expect it would get out of date unless the DevRel team has regular work items to keep this updated.

Reply:
Thank you. The DevRel team will maintain the index file.